CMakeLists.txt

```diff
@@ -67,7 +67,11 @@ ELSE()
   ADD_COMPILE_FLAG("-Wextra")
   ADD_COMPILE_FLAG("-Wno-unused-parameter")
   ADD_COMPILE_FLAG("-fno-omit-frame-pointer")
-  ADD_COMPILE_FLAG("-fPIC")
+  IF(WIN32)
+    ADD_COMPILE_FLAG("-D_GNU_SOURCE")
+  ELSE()
+    ADD_COMPILE_FLAG("-fPIC")
```

Comment:
and this flag is not available in MinGW environment: http://mingw.5.n7.nabble.com/Option-fPIC-not-supported-td18480.html